dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/task/DumpMetadataTask.java

```diff
@@ -52,20 +49,17 @@ public DumpMetadataTask(@Nonnull String format) {
   private DumpMetadataTask(
       @Nullable ConnectorArguments arguments,
       @Nonnull String format,
-      @Nullable List<UnderlyingConnector> connectors) {
+      @Nullable List<String> connectors) {
     super(ZIP_ENTRY_NAME);
     this.arguments = arguments;
     this.format = Preconditions.checkNotNull(format, "Format was null.");
-    this.connectors =
-        connectors != null
-            ? connectors.stream().map(UnderlyingConnector::name).collect(toImmutableList())
-            : null;
+    this.connectors = connectors != null ? ImmutableList.copyOf(connectors) : null;
   }
 
   public static DumpMetadataTask create(
       @Nonnull ConnectorArguments arguments,
       @Nonnull String format,
-      @Nonnull ImmutableList<UnderlyingConnector> connectors) {
+      @Nonnull ImmutableList<String> connectors) {
```

Comment:
Optional: `connectors` -> `connectorNames`

Reply:
Well, looks like it should be completely removed. Removed.